.travis.yml

```diff
@@ -1,14 +1,12 @@
 language: php
 
+dist: bionic
```

Comment:
we do not a specific `dist` for the test, about any linux dist will work

Reply:
Not true. Travis don't have all the PHP versions on each dist, and so it is good practice to pin the dist, so that when travis change the default dist, things don't break.

Reply:
Though, you are correct that the current default dist will work here, and we don't need to specify one. I am choosing to specify it though, for the reasons I said.